demos/card.html

```diff
@@ -127,210 +38,226 @@
+          <hr class="mdc-list-divider">
+          <section class="mdc-card__actions">
+            <div class="mdc-card__action-buttons demo-card__action-buttons--text">Rate this album</div>
+            <div class="mdc-card__action-icons">
+              <i class="material-icons mdc-card__action mdc-card__action--icon"
+                 tabindex="0"
+                 role="button"
+                 title="1 star">
+                star_border
+              </i>
+              <i class="material-icons mdc-card__action mdc-card__action--icon"
+                 tabindex="0"
+                 role="button"
+                 title="2 stars">
+                star_border
+              </i>
+              <i class="material-icons mdc-card__action mdc-card__action--icon"
+                 tabindex="0"
+                 role="button"
+                 title="3 stars">
+                star_border
+              </i>
+              <i class="material-icons mdc-card__action mdc-card__action--icon"
+                 tabindex="0"
+                 role="button"
+                 title="4 stars">
+                star_border
+              </i>
+              <i class="material-icons mdc-card__action mdc-card__action--icon"
+                 tabindex="0"
+                 role="button"
+                 title="5 stars">
+                star_border
+              </i>
             </div>
-          </div>
+          </section>
         </div>
-
       </section>
-
     </main>
+
+    <script src="/assets/material-components-web.js" async></script>
     <script>
-      // var formField = new mdc.formField.MDCFormField(document.querySelector('.mdc-form-field'));
-      // var cb = mdc.checkbox.MDCCheckbox.attachTo(document.querySelector('.mdc-checkbox'));
-      // formField.input = cb;
-      var demoWrapper = document.getElementById('demo-wrapper');
-      document.getElementById('toggle-rtl').addEventListener('change', function () {
-        if (this.checked) {
-          demoWrapper.setAttribute('dir', 'rtl');
-        } else {
-          demoWrapper.removeAttribute('dir');
-        }
+      demoReady(function() {
+        document.addEventListener('change', function(evt) {
+          var target = evt.target;
+          if (target.id !== 'toggle-rtl') {
+            return;
+          }
+
+          var docEl = document.documentElement;
+          if (target.checked) {
+            docEl.setAttribute('dir', 'rtl');
+          } else {
+            docEl.removeAttribute('dir');
+          }
+        });
+
+        [].forEach.call(document.querySelectorAll('.mdc-button'), function(surface) {
+          mdc.ripple.MDCRipple.attachTo(surface);
+        });
+
+        [].forEach.call(document.querySelectorAll('.mdc-icon-toggle'), function(iconToggle) {
+          mdc.iconToggle.MDCIconToggle.attachTo(iconToggle);
+        });
+
+        document.addEventListener('MDCIconToggle:change', function(evt) {
+          // TODO(acdvorak): Do this in icon-toggle
```

Comment:
I'm not sure it's a bug so much as a missing feature. We don't use the `title` attribute anywhere in our JS.

Personally, I think `title` attributes are **essential** on icons; if you don't already know what a hamburger icon means, you have no idea what's going to happen when you click on it.

Removed the TODO because we should fix this sort of thing during an accessibility audit.